workflows/transcriptomics/rnaseq-sr/rnaseq-sr.ga

```diff
@@ -23,289 +94,395 @@
+            "label": "Compute StringTie FPKM",
             "name": "Input parameter",
             "outputs": [],
             "position": {
-                "left": 245.88330078125,
-                "top": 926.4134391163916
+                "left": 406.9259176995931,
+                "top": 1559.4856968335116
             },
             "tool_id": null,
             "tool_state": "{\"parameter_type\": \"boolean\", \"optional\": false}",
             "tool_version": null,
             "type": "parameter_input",
-            "uuid": "05bdc4bb-4988-42e6-b281-cbb826b96c7b",
+            "uuid": "056f11f4-bdd9-4361-b037-2c0822320cc2",
             "when": null,
             "workflow_outputs": []
         },
-        "8": {
+        "10": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/4.9+galaxy1",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.23.4+galaxy2",
             "errors": null,
-            "id": 8,
+            "id": 10,
             "input_connections": {
-                "library|input_1": {
-                    "id": 0,
+                "single_paired|adapter_trimming_options|adapter_sequence1": {
+                    "id": 1,
                     "output_name": "output"
                 },
-                "library|r1|adapters_0|adapter_source|adapter": {
-                    "id": 1,
+                "single_paired|in1": {
+                    "id": 0,
                     "output_name": "output"
                 }
             },
             "inputs": [
                 {
-                    "description": "runtime parameter for tool Cutadapt",
-                    "name": "library"
+                    "description": "runtime parameter for tool fastp",
+                    "name": "single_paired"
                 }
             ],
-            "label": "Cutadapt (remove adapter + bad quality bases)",
-            "name": "Cutadapt",
+            "label": "remove adapters + bad quality bases",
+            "name": "fastp",
             "outputs": [
                 {
                     "name": "out1",
-                    "type": "fastqsanger"
+                    "type": "auto"
                 },
                 {
-                    "name": "report",
-                    "type": "txt"
+                    "name": "report_json",
+                    "type": "json"
                 }
             ],
             "position": {
-                "left": 500.61669921875,
-                "top": 339.134886401586
+                "left": 377,
+                "top": 106.21400000000001
             },
             "post_job_actions": {
+                "ChangeDatatypeActionout1": {
+                    "action_arguments": {
+                        "newtype": "fastqsanger"
+                    },
+                    "action_type": "ChangeDatatypeAction",
+                    "output_name": "out1"
+                },
```

Comment:
@mvdbeek here is the datatype change.

Reply:
Is this going to be fixed by https://github.com/galaxyproject/tools-iuc/pull/6499 ?

Reply:
Probably, I will check it on Monday.